packages/fluent-editor/src/modules/ai/index.ts

```
@@ -1,6 +1,68 @@
+  CLOSE_ICON,
+  COPY_ICON,
+  EDITOR_ICON,
+  INSERT_ICON,
+  MENU_CLOSE_ICON,
+  REBUILD_ICON,
+  REFRESH_ICON,
+  REPLACE_SELECT_ICON,
+  RICH_CONTENT_ICON,
+  RIGHT_ARROW_ICON,
+  SEND_BTN_ICON,
+  SHARE_ICON,
```

ESLint

[error] 'SHARE_ICON' is defined but never used.

(unused-imports/no-unused-imports)